neo/Cryptography/ECC/ECPoint.cs

```diff
@@ -47,13 +47,6 @@ public static ECPoint DecodePoint(byte[] encoded, ECCurve curve)
             int expectedLength = (curve.Q.GetBitLength() + 7) / 8;
```

Comment:
This could be done in another PR, right? It is just a clean on the ECPoint, yes?

Reply:
If we don't change this, the tests will be broken.